sympy/printing/str.py

```diff
@@ -728,6 +728,10 @@ def _print_Tr(self, expr):
         #TODO : Handle indices
         return "%s(%s)" % ("Tr", self._print(expr.args[0]))
 
+    def _print_DifferentialOperator(self, expr):
+        print_str = expr.str()
```

Comment:
Put the actual printer here. Also, the `str` printer should always be valid Python, which recreates the class (use the pretty printer for fancy printing). And make sure to use `self._print` to recursively print subexpresions.